### DISCUSSION TITLE
新的表格识别算法-SLANet-LCNetV2，推理模型结果没法复现

### DISCUSSION THREAD

**Discussion by zhujun5164** [PaddlePaddle/PaddleOCR#13333] [Q&A]
我测试了下运行模型的测试代码，但是从输出结果来看和展示的存在明显区别。

python table/predict_structure.py --table_model_dir=../checkpoint/table/ --table_char_dict_path=../ppocr/utils/dict/table_structure_dict_ch.txt --image_dir=docs/table/table.jpg --output=../output/table_slanet_lcnetv2 --use_gpu=False --benchmark=True --enable_mkldnn=True --table_max_len=512

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/49580602/c05a9637-cd85-41db-848e-0d0891628386)
![table](https://github.com/PaddlePaddle/PaddleOCR/assets/49580602/1b5809f1-26d9-4103-853c-65cbc042f9c2)

**Comment by gphmath** [comment]
--table_char_dict_path=../ppocr/utils/dict/table_structure_dict_ch.txt
这个dict替换成：
--table_char_dict_path=../ppocr/utils/dict/table_structure_dict.txt
就可以复现官网这个效果了
顺便想确认下，是不是这样：目前开源的是一个英文表格模型，所以要用table_structure_dict.txt。另外有一个中文模型，是使用table_structure_dict.txt的，但是这个中文模型没有开放下载？

**Reply by GreatV** [reply]
不是的，这个只是typo。

**Comment by Andy197527** [comment]
请问，这个百度的表格识别模型怎么样子？为什么我调用这个模型出来的效果并不好，有一些错误。有其他的表格识别模型推荐吗？
